packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts

```diff
@@ -276,10 +277,14 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
     }
 
     private createActionElement({ id, label, type, action }: ContextMenuAction): HTMLElement {
+        let button: HTMLElement;
         if (id && this.registry.isDisabled(id)) {
-            return this.createDisabledElement(label);
+            button = this.createDisabledElement(label);
+        } else {
+            button = this.createButtonElement(label);
```

Comment:
nit: could merge `createButtonElement` and `createDisabledElement` as they're so similar? Then add the `ariaDisabled` in here if disabled?

Reply:
Done.